scripts/dev_scripts/integration_tests.sh

```diff
@@ -636,6 +638,32 @@ POLICY_EXPECTED=$WORKSPACE/tests/policy_engine/expected_results/policy_report.js
 $RUN_POLICY -f $POLICY_FILE -d "$WORKSPACE/output/macaron.db" || log_fail
 check_or_update_expected_output $COMPARE_POLICIES $POLICY_RESULT $POLICY_EXPECTED || log_fail
 
+# Testing the VSA generation feature
+# Running Macaron without config files
+echo -e "\n=================================================================================="
+echo "Run integration tests for VSA generation"
+echo -e "==================================================================================\n"
+TEST_CASE_DIR="$WORKSPACE/tests/vsa/integration/github_slsa-framework_slsa-verifier"
+OUTPUT_DIR="$TEST_CASE_DIR/output"
+
+rm -rf "$OUTPUT_DIR"  # Make sure we regenerate a fresh database every time.
+macaron --output "$OUTPUT_DIR" analyze \
+    --repo-path "https://github.com/slsa-framework/slsa-verifier" \
```

Comment:
Analyzing `slsa-verifier` in general takes a long time because its release artifacts are quite large and that's why we cache the results and run the policy engine tests without running the `analyze` command again. Note that we already analyze `slsa-verifier` [here](https://github.com/oracle/macaron/blob/8334460935909d7fa8b05ee32fad02107185176d/scripts/dev_scripts/integration_tests.sh#L613). How about using the existing results?

Also, to keep the script consistent, we should either use `$RUN_MACARON` here and follow the same naming conventions for the resources and expected results or change the rest of the script to be consistent with the VSA test.

Reply:
If you think it's OK to reuse the cached result, then I can move this test outside of the isolated directory. My initial intention is to have this integration test isolated to make sure there is only one software component in the database. Meanwhile, there are already some unit tests that take care of "different states of the database", so I think using the cached result is fine.
`$RUN_MACARON` not being used was a side-effect of the initial design of the test to be in a separate directory. The `$RUN_MACARON` command fixes the location of the `output` directory, whereas my first intention was to have a separate `output` directory altogether just for this test.

Reply:
I agree that ideally we should avoid using cached results, but this is purely to reduce the CI runtime. I think it's OK to use the cached result and add a comment to clarify that.

Reply:
See my adjustments in a272ff9c0cc09840ac87a9e9b1a402a1733a3172.